Source/WebKit/UIProcess/Cocoa/ExtensionCapabilityGranter.mm

```diff
@@ -51,24 +53,31 @@
 }
 
 #if USE(EXTENSIONKIT)
-static RetainPtr<_SEGrant> grantCapability(_SECapability *capability, _SEExtensionProcess *process)
+static RetainPtr<BEProcessCapabilityGrant> grantCapability(BEProcessCapability *capability, ExtensionProcess process)
 {
     ASSERT(capability);
-    if (!capability || !process)
+    if (!capability)
         return nil;
 
     NSError *error = nil;
-    _SEGrant *grant = [process grantCapability:capability error:&error];
+    BEProcessCapabilityGrant* grant = nil;
```

Comment:
Nit: space goes before asterisk.

Reply:
Thanks for reviewing! I will address these issues in the upcoming patch.